ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteImporter.java

```diff
@@ -101,7 +103,7 @@ private static EntsoeGeographicalCode getRegionalGeographicalCode(Substation sub
         return res == EntsoeGeographicalCode.DE ? null : res;
     }
 
-    private static void createBuses(UcteNetworkExt ucteNetwork, UcteVoltageLevel ucteVoltageLevel, VoltageLevel voltageLevel) {
+    private static void createBuses(UcteNetworkExt ucteNetwork, UcteVoltageLevel ucteVoltageLevel, VoltageLevel voltageLevel, Reporter reporter) {
```

Comment:
Is an importer supposed to be thread-safe ? If not, would it be better to store the reporter in a field of the importer object so we don't have to pass it around explicitly everywhere ?

Reply:
It is supposed to be thread safe

Reply:
And it is also need as UcteImporter is @Autoservice